components/Chips/tests/unit/ChipsTests.m

```diff
@@ -394,4 +411,38 @@ - (void)testChipsWithDeleteEnabled {
   XCTAssertEqual(controlViewCount, (NSUInteger)1);
 }
 
+- (void)testChipViewDynamicTypeBehavior {
+  // Given
+  FakeMDCChipView *chipView = [[FakeMDCChipView alloc] init];
```

Comment:
If we plan on testing all the different size categories maybe start a new file for dynamic type test and check all categories?

Reply:
It is a good point, the worry I have is we have a separate DT testing for ensure all values are correct on the scaler level. So for this component, I felt couple example size categories would be enough. Open for discussion.

Reply:
Sounds good.